src/EditorFeatures/CSharpTest2/Recommendations/DefaultKeywordRecommenderTests.cs

```diff
@@ -312,5 +312,19 @@ public async Task TestInTypeParameterConstraint_InExplicitInterfaceImplementatio
     }
 }");
         }
+
+        [Fact]
+        public async Task TestAfterLambdaOpenParen()
+        {
+            await VerifyKeywordAsync(
+@"var lam = ($$");
+        }
+
+        [Fact]
+        public async Task TestAfterLambdaComma()
+        {
+            await VerifyKeywordAsync(
+@"var lam = (int i, $$");
```

Comment:
Consider also testing `var lambda = (int i = $$` (should offer `default` and `null`)

Reply:
Thanks, added: 87514b64a54bae17e8b770127e80b15fd8e67360